package.json

```diff
@@ -81,7 +82,7 @@
     "st": "^2.0.0",
     "watchify": "^3.11.1",
     "wd": "^1.11.2",
-    "webidl2js": "^13.0.0"
+    "webidl2js": "pmdartus/webidl2js#pmdartus/add-processor"
```

Comment:
Don’t&nbsp;forget to&nbsp;use the&nbsp;production **WebIDL2JS**&nbsp;release before&nbsp;merging.